src/mocks/@LanOnasis/memory-sdk/index.ts

```
@@ -0,0 +1,50 @@
+
+export interface MemoryClient {
+  createMemory(data: any): Promise<any>;
+  searchMemories(query: string): Promise<any>;
+  updateMemory(id: string, data: any): Promise<any>;
+  deleteMemory(id: string): Promise<any>;
+  upsert(data: any): Promise<any>;
+  search(query: any): Promise<any>;
+  rerank?(data: any): Promise<any>;
+}
+
+export class MemoryClient {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)